aiidalab/registry/releases.py

```diff
@@ -30,15 +30,62 @@ def _split_release_line(url):
     return url, None
 
 
+def _get_tags(repo, rev, rev_selection):
```

Comment:
This is a non-trivial function, could you please add a docstring? Especially explaining the input parameters and the output. Adding types would help understanding as well.

Reply:
I change `rev` to `branch` as the check happened in the function.